fixtures/fixtures.js

```diff
@@ -90,12 +102,27 @@ Fixtures.apply = function (mock) {
     mock.onGet(/\/traders\/2\/vouchers/).reply(200, this.traderVouchers["2"]);
 
     mock.onPost('/vouchers').reply(function (request) {
-        // returns a success regardless!
-        // TODO : better emulation of server side validated responses;
+        const requestPayload = JSON.parse(request.data);
+
+        const success = [];
+        const fail = [];
+        const invalid = [];
+
+        requestPayload.vouchers.forEach((voucherNumber) => {
```

Comment:
where are you getting the voucher numbers from? If we are on fixtures - I think we just use the fixtures - not actually take input. @charlesstrange2 What do you think?